.github/workflows/playwright.yml

```diff
@@ -4,6 +4,8 @@ on:
     branches: [main, beta]
   pull_request:
     branches: [main, beta]
+env:
+  NEXT_PUBLIC_BASE_URL: http://localhost:3000
```

Comment:
When this GH action was running in this PR initially, `NEXT_PUBLIC_BASE_URL` ended up being `'undefined'` not `undefined` which caused the based URL of OG images to be `'undefined'`. e.g.

```typescript
const ogImage = new URL(
  getWorkspaceOgImage(data as Workspace, range),
  process.env.NEXT_PUBLIC_BASE_URL || "http://localhost:3000"
);
```